src/JustSaying/AwsTools/MessageHandling/SnsPolicy.cs

```diff
@@ -1,3 +1,4 @@
+#pragma warning disable 618
```

Comment:
`Amazon.Auth` is now obsolete: https://github.com/aws/aws-sdk-net/commit/6f9f7aa20e92180ecbf062d8fba8f67dbcc68add